packages/runtime/src/next/api-handler/handlers/proxy-draft-mode.ts

```diff
@@ -1,7 +1,9 @@
 import { NextApiRequest, NextApiResponse } from 'next'
 import { NextRequest, NextResponse } from 'next/server'
 import { P, match } from 'ts-pattern'
-import { cookies, draftMode } from 'next/headers'
+
+import { cookies, enableDraftMode } from '../../lib/next'
```

Comment:
Compatibility layer for Next.js 15, see https://github.com/makeswift/makeswift/pull/857/files#diff-90b2f7383e5a1a78d0640d9e7f60d69143d095872d61c0da07309975c898d1dbR1